### ISSUE TITLE
void WhenReady 5.3

### ISSUE THREAD

**Issue by CodenameReality** [immutable/unreal-immutable-sdk#46]
For 5.3 compilation, change in .h:

  void WhenReady(UserClass *Object,
                 typename FImmutableSubsystemReadyDelegate::FDelegate::TMethodPtr<UserClass> Func);
  // ::TUObjectMethodDelegate<UserClass>::FMethodPtr
  
  
  In cpp:
  template <class UserClass>
void UImmutableSubsystem::WhenReady(
    UserClass *Object, typename FImmutableSubsystemReadyDelegate::FDelegate::TMethodPtr<UserClass> Func) {
    // ::TUObjectMethodDelegate<UserClass>::FMethodPtr
  OnReady.AddUObject(Object, Func);
}

**Comment by DooblyNoobly** [comment]
this is what worked for me:

ImmutableSubsystem.h:
```c++
template<class UserClass>
void WhenReady(UserClass *Object, typename FImmutableSubsystemReadyDelegate::FDelegate::TMethodPtr<UserClass> Func);
```

ImmutableSubsystem.cpp:
```c++
void UImmutableSubsystem::WhenReady(UserClass* Object, FImmutableSubsystemReadyDelegate::FDelegate::TMethodPtr<UserClass> Func)
{
    OnReady.AddUObject(Object, Func);
}
```

**Comment by YermekG** [comment]
@CodenameReality, The issue has been fixed in #52, please update source code of your plugin. Changes mentioned by @DooblyNoobly are introduced in the fix.
